packages/extract-react-types/src/utils.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
This eslint rule creates lot of noise in repos. We can disable it altogether.

Reply:
Yeah i think we should kill this one too!